md/release-notes.md

```diff
@@ -85,10 +107,15 @@ Jasper connector has been removed from provided connectors in Bonita Studio. If
 * Export the Jasper connector from a previous Studio version
 * Download the connector from the [community website](https://community.bonitasoft.com/project/bonita-connector-jasper)
   Then just import the connector using the `Development > Connectors > Import connector...` menu.
+  
 
 ## API behavior change
 
-### Add comment in an open case
+### UI Designer URL change
+The UI Designer webapp embedded in Bonita Studio is now deployed with the name "bonita" (instead of "designer") in its own container (different from the one of Bonita Engine and Portal). As a result, it is now accessible through the URL http://localhost:<designer_port>/bonita instead of http://localhost:<tomcat_port>/designer.
+This name update was a requirement in order for the UI Designer preview to continue working with bonita portal resources URLs starting with "/bonita".
```

Comment:
This was an internal requirement. I feel its not necessary for any customer to know that...

Reply:
Pablo's feedback shows that customers and consultants hard-coded the URL. This will be broken in 7.7. We need to give a clue to what has been changed, at least in the documentation.